gdc_client/parcel/segment.py

```diff
@@ -317,7 +317,7 @@ def wait_for_completion(self):
                     this_size = interval.end - interval.begin
                     self.size_complete += this_size
                     since_save += this_size
-                    self.pbar.update(this_size)
+                    self.pbar.update(self.size_complete)
```

Comment:
The previous code has a `try-catch` here because `pbar.update(n)` will raise an error if `n > pbar.max_value`.  We should either add that `try-catch` back or have some logic to make sure we're updating with a valid value.  Having the progresss bar abort a transfer would be undesirable.